docs/docs/reference/shards/lisp/misc.md

```diff
@@ -33,44 +33,92 @@ This is useful for cases where you need multiple shards (to transform your data)
 
 ## apply
 
-```clojure linenums="1"
+```
+(apply f args) (apply f x args) (apply f x y args) (apply f x y z args)(apply f a b c d & args)
+```
+
+Applies fn f to the argument list formed by prepending intervening arguments to args.
 
+```clojure linenums="1"
+(apply str ["cherry " "banana " "apple"])
+;;=> "cherry banana apple"
 ```
 
 ## atom
 
-```clojure linenums="1"
+```
+(atom x)
+```
 
+  Creates and returns an Atom with an initial value of x and zero or more options (in any order).
```

Comment:
Maybe add a brief description of what an atom is

Reply:
Is explain good?

Reply:
A bit, I'm asuming you'd use these to store data, so it's like this:
```clj
(def atom (atom 1))
(reset! atom 2)
```
?

Reply:
alright, I improved with example.